### DISCUSSION TITLE
Improved notification behavior

### DISCUSSION THREAD

**Discussion by gilcu3** [d99kris/nchat#207] [Ideas]
Currently, receiving a notification in a chat makes future notifications in the same chat disabled until the chat is opened. This does not match the standard behavior of chat apps, where every message is notified to the user. Is there a specific reason for this, and any way to change it?

Also, when starting `nchat`, all messages received since it was last opened generate notifications. I think a more sane behavior would be to not notify about anything on startup, then activate notifications right after. Is it possible to do so currently?

**Comment by d99kris** [comment]
Hi @gilcu3 - I've changed this topic to an enhancement request. I think you bring up some valid points. I'll try look into them when I find some time.

**Comment by d99kris** [comment]
Hi again @gilcu3 - the suggested improvements have been implemented now, in 67ccda3 and a02c648 respectively. Thanks for the suggestions!
